leaf-ui/js/onFunctions.js

```diff
@@ -832,10 +832,11 @@ function createIntention(cell) {
  * 
  * @param {joint.dia.Cell} cell
  */
-function createActor(cell) {
+function createActor(cell) {//TODO: right now there are two parameters the actor model in the joint.extensions file that hold the same information (attrs.name & actorName), find a way for actor inspector to be able to access attrs.name in the template script so that actorName is not needed
 	var name = cell.attr('.name/text') + "_" + Actor.numOfCreatedInstances;
 	var actor = new Actor(name);
     cell.attr(".name/text", name);
+    cell.set('actorName', name);
```

Comment:
You could potentially make a function inside the actor cell that updates the actorName and the .name/text param at the same time? So the function only has to be called once

Reply:
Is it necessary? Because we would only call it once.

Reply:
I decided not to create a function for that.

Reply:
Okay, we may call it more times in the future (see createActor() function) but for now this is fine! We can always add a function if we end up using it a lot